modules/ting_search_carousel/ting_search_carousel.module

```diff
@@ -1,11 +1,13 @@
 <?php
+
 /**
  * @file
  * Ting search carousel module main file.
  */
 
 // The number of covers returned in each chunk.
-define('TING_SEARCH_CAROUSEL_CHUNK_SIZE', 12);
+define('TING_SEARCH_CAROUSEL_CHUNK_SIZE', 8);
```

Comment:
So we by default load/show less items per chunk with the new carousel? Is that intentional?

Reply:
Yes, as the new carousel with DDB Basic theme at most shows 4 items and steps size is calculated to 2. So this seams to be the optional number of chunks to load in relation to performance and that you can click two times "next" in the carousel before having to wait for a ajax load of the next chunk.

Reply:
:+1: